servicetalk-tcp-netty-internal/src/main/java/io/servicetalk/tcp/netty/internal/TcpServerBinder.java

```diff
@@ -87,7 +97,9 @@ public static <CC extends ConnectionContext> Single<ServerContext> bind(SocketAd
             final ReadOnlyTcpServerConfig config, final boolean autoRead, final ExecutionContext<?> executionContext,
             @Nullable final InfluencerConnectionAcceptor connectionAcceptor,
             final BiFunction<Channel, ConnectionObserver, Single<CC>> connectionFunction,
-            final Consumer<CC> connectionConsumer) {
+            final Consumer<CC> connectionConsumer,
+            @Nullable final EarlyConnectionAcceptor earlyConnectionAcceptor,
+            @Nullable final LateConnectionAcceptor lateConnectionAcceptor) {
```

Comment:
Even though this is an internal module, for 0.42.x we try to be extra cautious. Because it's a public method, let's deprecate the existing overload and create a new one.